.gitignore

```diff
@@ -8,7 +8,7 @@ nupkg/*
 .generated/
 *.*.import
 TestResults/
-gdunit4_testadapter/
+# gdunit4_testadapter/
```

Comment:
We need to clarify how to proceed with the installed test runner.
Actually, we store it into the repository because of install and rebuild issues at test adapter execution.